prow/cmd/crier/README.md

```diff
@@ -34,6 +34,8 @@ You need to specify following labels in order for pubsub reporter to report your
 | `"prow.k8s.io/pubsub.topic"`   | The [topic](https://cloud.google.com/pubsub/docs/publisher) of your pubsub message                                                        |
 | `"prow.k8s.io/pubsub.runID"`   | A user assigned job id. It's tied to the prowjob, serves as a name tag and help user to differentiate results in multiple pubsub messages |
 
+The service account used by crier will need to have `pubsub.topics.publish` permission in the project where pubsub channel lives.
```

Comment:
In the spirit of @fejta's suggestion to use something that is easy copy-pasta for `gcloud`
```suggestion
The service account used by crier will need to have `pubsub.topics.publish` permission in the project where pubsub channel lives, e.g. by assigning the `roles/pubsub.publisher` IAM role
```

Also.. does it need to be in the _project_ or would a more narrowly-scoped `gcloud pubsub topics set-iam-policy` work?